css/main.css

```diff
@@ -96,7 +96,8 @@ nav li:hover ul {
 nav li ul li {
     display: inline-block;
     margin: 0px 3px;
```

Comment:
For future reference, if you use 0 you don't need to give a unit, so you can just say `margin: 0 3px`